.travis.yml

```diff
@@ -21,9 +21,14 @@ matrix:
   allow_failures:
   - rust: nightly
 
+
+before_script:
+  - rustup component add clippy
+
 script:
   - cargo build --all
   - cargo test --all
+  - ./clippy.sh
```

Comment:
We could use cargo config to make an alias: [example](https://github.com/rust-analyzer/rust-analyzer/blob/master/.cargo/config).